### PR TITLE
fix: Updates to cds beta

### PR DESCRIPTION
**Pull request by daniejoh**
https://confluence.ecmwf.int/display/CKB/Please+read%3A+CDS+and+ADS+migrating+to+new+infrastructure%3A+Common+Data+Store+%28CDS%29+Engine

ECMWF is changing its api. This pr changes the data-pipeline to use this new endpoint and new credentials.

### CODE REVIEW

**Comment by daniejoh** [comment]
Depends on: https://github.com/openearthplatforminitiative/infrastructure/pull/57

File: deployment/dagster/values.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ dagster-user-deployments:
     - name: "data-pipelines"
       image:
         repository: "ghcr.io/openearthplatforminitiative/data-pipelines"
-        tag: latest
+        tag: dan-test-2
```
Comment (kstigen) [inline deployment/dagster/values.yaml:7]: Need to change this back to latest.

File: Dockerfile [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
+    libproj-dev \
+    gdal-bin \
+    libgdal-dev
 
 ENV POETRY_VIRTUALENVS_CREATE=false
 WORKDIR /opt/dagster/app
 RUN pip install poetry
 COPY pyproject.toml poetry.lock /opt/dagster/app/
+
+RUN poetry config --local installer.no-binary rasterio,fiona
```
Comment (kstigen) [inline Dockerfile:20]: What does this do?

Comment (daniejoh) [reply Dockerfile:20]: Will add a comment. It ensures that it does not install the binaries for rasterio and fiona. They are not needed, and makes the installation crash on my m1 mac.

**Review by kstigen — APPROVED** [review]

**Pull request merged**